api/pom.xml

```diff
@@ -83,6 +83,12 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>5.10.3</version>
```

Comment:
We should use the `junit.version` property here. Also, now that we’re using the three JUnit dependencies, maybe we should replace them with the single dependency `junit-jupiter`?